util/sparse/punch_linux.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//go:build linux
+
+// Package sparse contains some helpful generic sparse file functions.
+package sparse
+
+import (
+	"os"
+	"syscall"
+
+	"golang.org/x/sys/unix"
+)
+
+// punchAt for linux
```

Comment:
Nit: I find this comment kinda useless. It's obviously `punchAt` and it's obviously for `linux`. I would just have no comment unless there was something noteworthy about Linux's behavior in particular.